x-pack/test/api_integration/deployment_agnostic/apis/observability/ai_assistant/knowledge_base/knowledge_base_change_model_from_elser_to_e5.spec.ts

```diff
@@ -44,6 +44,8 @@ export default function ApiTest({ getService }: DeploymentAgnosticFtrProviderCon
   type KnowledgeBaseEsEntry = Awaited<ReturnType<typeof getKnowledgeBaseEntriesFromEs>>[0];
 
   describe('Knowledge base: when changing from ELSER to E5-like model', function () {
+    // see details: https://github.com/elastic/kibana/issues/220513
+    this.tags(['failsOnMKI']);
```

Comment:
Probably this should also be skipped in ECH.

```suggestion
    this.tags(['skipCloud']);
```